ports/blas/vcpkg.json

```
@@ -1,9 +1,12 @@
 {
   "name": "blas",
-  "version-string": "1",
-  "port-version": 1,
+  "version-date": "2022-03-31",
   "description": "Metapackage for packages which provide BLAS",
```

```suggestion
  "description": "Metapackage for packages which provide BLAS",
  "license": null,
```
Unimportant change. Will be added after team review.